src/main/scala/com/hortonworks/spark/sql/hive/llap/HS2JDBCWrapper.scala

```diff
@@ -20,11 +20,15 @@ package com.hortonworks.spark.sql.hive.llap
 import java.net.URI
 import java.sql.{Connection, DatabaseMetaData, Driver, DriverManager, ResultSet, ResultSetMetaData, SQLException}
 
```

Comment:
Our current master is 2.1. thus, we need to land it to the 2.2 and 1.6 as well. I can verify the difference.